### PR TITLE
Update scala 2.13.0-M5 to 2.13.0-RC2

### PR DESCRIPTION
**Pull request by Philippus**

### CODE REVIEW

File: src/main/scala/grizzled/file/Implicits.scala [hunk 1]
```diff
@@ -34,7 +34,7 @@ object Implicits {
       if (file.exists)
         Success(true)
       else
-        Failure(new FileDoesNotExistException(s""""$file" does not exist."""))
+        Failure(new FileDoesNotExistException(s""""${file.toString}" does not exist."""))
```
Comment (SethTisue) [inline src/main/scala/grizzled/file/Implicits.scala]: curious about this and the other similar diffs, why is this necessary/desirable?

Comment (Philippus) [reply src/main/scala/grizzled/file/Implicits.scala]: The `[wartremover:StringPlusAny] Implicit conversion to string is disabled`-wart triggers on scala 2.13.0-RC1, which maybe it shouldn't? On scala 2.12.8 it does not.

Comment (SethTisue) [reply src/main/scala/grizzled/file/Implicits.scala]: ah. hmmm....

I think I personally wouldn't expect or want the wart to trigger inside a string interpolator, but that's just a hunch, not sure what the wart's author's intentions might have been

Comment (Philippus) [reply src/main/scala/grizzled/file/Implicits.scala]: I filed an issue here: https://github.com/wartremover/wartremover/issues/447

**Comment by bmc** [comment]
Merged. I'll release an update soon—this weekend, if not before. (I've been swamped.)
